azure-functions/acbs-function/api-dtfs-cental-api.js

```diff
@@ -0,0 +1,45 @@
+const axios = require('axios');
+
+const centralAPI = process.env.DTFS_CENTRAL_API;
+
+require('dotenv').config();
```

Comment:
`process.env` is dependant on `dotenv`. so this line should be before `const centralAPI = ...`